internal/dashboard/model/sql/plan_info.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (c) 2023 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+
+package sql
+
+type PlanCategory string
+
+const (
+	PlanCategoryLocal       PlanCategory = "local"
+	PlanCategoryRemote      PlanCategory = "remote"
+	PlanCategoryDistributed PlanCategory = "distributed"
+)
+
+type PlanMeta struct {
+	PlanHash      string       `json:"planHash" binding:"required"`
+	Category      PlanCategory `json:"category" binding:"required"`
+	MergedVersion int          `json:"mergedVersion" binding:"required"`
+	GeneratedTime int64        `json:"generatedTime" binding:"required"`
+}
+
+type PlanStatistic struct {
+	PlanMeta
```

Comment:
`json:",inline"` should be added to this line if the `PlanMeta` is expected to be included in response json.